externals/kyuubi-chat-engine/src/main/scala/org/apache/kyuubi/engine/chat/provider/ChatGPTProvider.scala

```diff
@@ -60,6 +60,8 @@ class ChatGPTProvider(conf: KyuubiConf) extends ChatProvider {
     new OpenAiService(api)
   }
 
+  private var sessionUser: Option[String] = None
```

Comment:
Seems not right to have a shared `sessionUser` crossing sessions in ChatProvider instance.